src/lib/components/archiveProject.svelte

```diff
@@ -275,6 +317,42 @@
     </svelte:fragment>
 </Modal>
 
+<!-- Delete Confirmation Modal -->
+<Modal
+    size="s"
+    bind:show={showDeleteModal}
+    title="Delete project"
+    onSubmit={confirmDelete}
+    bind:error={deleteError}>
+    <svelte:fragment slot="description">
```

Comment:
is the copy already checked by Design?

Reply:
@HarshMN2345 may be we refactor to use the same delete model in settings and here too?